WalletWasabi/Blockchain/Transactions/Summary/KnownInput.cs

```diff
@@ -0,0 +1,23 @@
+using NBitcoin;
+using WalletWasabi.Blockchain.TransactionOutputs;
+
+namespace WalletWasabi.Blockchain.Transactions.Summary;
+
+public class KnownInput : IInput
+{
+	private readonly SmartCoin _coin;
+	private readonly Network _network;
+
+	public KnownInput(SmartCoin coin, Network network)
+	{
+		_coin = coin;
+		_network = network;
+	}
+
+	public virtual Money Amount => _coin.Amount;
+
+	public BitcoinAddress Address => _coin.ScriptPubKey.GetDestinationAddress(_network)!;
+	public double Anonscore => _coin.HdPubKey.AnonymitySet;
```

Comment:
You forgot to display this. Please do that.

Reply:
Oops. Overlooked that. 
Fixed now.